lib/Runtime/Library/JavascriptLibrary.h

```diff
@@ -381,15 +382,22 @@ namespace Js
 
 
         JavascriptSymbol* symbolMatch;
+        JavascriptSymbol* symbolReplace;
         JavascriptSymbol* symbolSearch;
         JavascriptSymbol* symbolSplit;
 
         UnifiedRegex::RegexPattern * emptyRegexPattern;
+        int regexConstructorSlotIndex;
         JavascriptFunction* regexExecFunction;
+        int regexExecSlotIndex;
         JavascriptFunction* regexFlagsGetterFunction;
+        int regexFlagsGetterSlotIndex;
         JavascriptFunction* regexGlobalGetterFunction;
+        int regexGlobalGetterSlotIndex;
         JavascriptFunction* regexStickyGetterFunction;
+        int regexStickyGetterSlotIndex;
         JavascriptFunction* regexUnicodeGetterFunction;
+        int regexUnicodeGetterSlotIndex;
```

Comment:
I was thinking about that too. Readability now wins (dbd1646e6870c92818af6fef48cb2702952289cc).